batcher/aligned-batcher/src/lib.rs

```diff
@@ -1786,7 +1795,12 @@ impl Batcher {
             ETHEREUM_CALL_MAX_RETRY_DELAY,
         )
         .await
-        .map_err(|e| BatcherError::BatchUploadError(e.to_string()))
+        .map_err(|e| BatcherError::BatchUploadError(e.to_string()))?;
+
+        self.metrics
+            .s3_latency
+            .set(start.elapsed().as_micros() as i64);
```

Comment:
send value even if it fails should visualize a peak in latency

Reply:
Why would it be different than a success case after the max number of retries have been performed?
All errors thrown by this function are transient, so it can only return an error at this point if all retries failed. I think the peak would be the same if that last retry were successful instead of a failure.

Reply:
> Why would it be different than a success case after the max number of retries have been performed?
All errors thrown by this function are transient, so it can only return an error at this point if all retries failed. I think the peak would be the same if that last retry were successful instead of a failure. @JulianVentura 

The problem here is the "?" at the end of the function call. I will refactor this.